volatility/framework/plugins/windows/vaddump.py

```diff
@@ -36,14 +36,33 @@ def get_requirements(cls) -> List[interfaces.configuration.RequirementInterface]
                 requirements.PluginRequirement(name = 'vadinfo', plugin = vadinfo.VadInfo, version = (1, 0, 0)),
                 ]
 
+    @staticmethod
+    def vad_dump(vad, proc_layer):
```

Comment:
Another convention is that we usually pass `context` and `layer_name` rather than passing in a full layer.  I think this is because if the method ever needs to be extended in the future, having the context means you've got everything, whereas having just the layer makes it slightly more troublesome to get other things you might need (such as symbol tables, etc).  I think since this is core code, and we need to be careful of interfaces changing in the future, we should try to be consistent, so this is a change I think has to happen.

I'd expect the final signature to be something along the lines of:
```
def vad_dump(cls, context: interfaces.context.ContextInterface, layer_name: str, vad: interfaces.objects.ObjectInterface) -> bytes:
```